### PR TITLE
Add edf

### LINKED ISSUE #27: [Feature] Support plot_edf to show the objective value EDF (empirical distribution function)
# Feature Request

Porting `plot_edf()` to TypeScript.
https://github.com/optuna/optuna/blob/v2.4.0/optuna/visualization/_edf.py

### PR DESCRIPTION
**Pull request by chenghuzi**
## Contributor License Agreement

This repository (``optuna-dashboard``) and [Goptuna](https://github.com/c-bata/goptuna) share common code.
This pull request may therefore be ported to Goptuna.
Make sure that you understand the consequences concerning licenses and check the box below if you accept the term before creating this pull request.

* [x] I agree this patch may be ported to [Goptuna](https://github.com/c-bata/goptuna) by other Goptuna contributors.

## Reference Issues/PRs

Close #27 

## What does this implement/fix? Explain your changes.

Added edf chart, but only for single study case.

### CODE REVIEW

File: optuna_dashboard/static/components/GraphEdf.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    return
+  }
+
+  const target_name = "Objective Value"
+
+  const _target = (t: Trial): number => {
+    return t.values![0]
+  }
+
+  const target = _target
```
Comment (c-bata) [inline optuna_dashboard/static/components/GraphEdf.tsx:32]: It looks that you can remove `_target` variable here.

**Review by c-bata — COMMENTED** [review]
Thank you for your pull request. The changes are easy to read and work as expected.

ScreenShot 2021-04-05 10 22 50

I put a nit-picking review comment.

File: optuna_dashboard/static/components/GraphEdf.tsx (hunk 1 above)
Comment (chenghuzi) [reply optuna_dashboard/static/components/GraphEdf.tsx:32]: Yeah, but I added it here to remind me of a potential future multi-objective case, in which we may need to select between different `_target`s. So could we leave it as it is now?

Comment (c-bata) [reply optuna_dashboard/static/components/GraphEdf.tsx:32]: OK! I see.

**Review by c-bata — APPROVED** [review]
Thank you for your pull request!

**Pull request merged**